src/FluentResults/Results/Result.cs

```diff
@@ -90,8 +90,8 @@ public override string ToString()
         {
             return result.ToResult<TValue>();
         }
-
-        public static implicit operator Result(Result<TValue> result)
+        
+        public static implicit operator Result<TValue>(TValue result)
```

Comment:
Iam not a big fan of the implicit operator of object to Result<T>. Implicit operator from object are not recommended in general.

Reply:
> Sorry for the delay. I had a look at the changes and like a lot. My aim is to prevent breaking changes. Would you implement my suggestions or should I bring this pr to an end?

Yes, I will implement your changes 😄 
And I would like to talk about the implicit operator, but later)